client.go

```diff
@@ -14,6 +14,9 @@ type Client struct {
 	// Debug enables debugging output through log package
 	Debug bool
 
+	// HTTPClientDebug enables debugging output through log package
+	HTTPClientDebug bool
+
```

Comment:
two debug flags with same documentation  make confused.
I think we should rename 
```
Debug --> Verbose
HTTPClientDebug --> Trace
````
and  document both flags more clearly.